docs/terminal/eastsale.md

```diff
@@ -4,7 +4,226 @@
+| Alpha         | Northern Lane and Lakes Lane               | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A059 FT |
+| Bravo         | Lakes Lane and Eastern Lane                | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A030 FT |
+| Charlie       | Eastern Lane and Bass Lane (ESL NDB)       | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A019 FT |
+| Delta         | Bass Lane and Southern Lane                | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A024 FT |
+| Echo          | Southern Lane and Strezlecki Lane          | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A039 FT |
+| Foxtrot       | Strezlecki Lane and Western Lane           | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A070 FT to F430 (20 NM to 35 NM) | A066 FT |
+| Golf          | Western Lane and Avon Lane                 | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM); 7000 FT to FL430 (20 NM to 35 NM) | A070 FT |
+| Hotel         | Avon Lane and Northern Lane                | 12 NM to 35 NM   | SFC to F430 (12 NM to 20 NM); A050 FT to F430 (20 NM to 35 NM) | A070 FT |
+| Sierra        | Northern Lane and Lakes Lane               | 35 NM to 50 NM   | A050 FT to FL430                                      | A065 FT |
+| Tango         | Lakes Lane and Eastern Lane                | 35 NM to 50 NM   | A050 FT to FL430                                      | A035 FT |
+| Uniform       | Eastern Lane and Bass Lane                 | 35 NM to 50 NM   | A050 FT to FL430                                      | A015 FT |
+| Victor        | Bass Lane and Southern Lane                | 35 NM to 50 NM   | A050 FT to FL430                                      | A017 FT |
+| Whiskey       | Southern Lane and Strezlecki Lane          | 35 NM to 50 NM   | A050 FT to FL430                                      | A039 FT |
+| X-ray         | Strezlecki Lane and Western Lane           | 35 NM to 50 NM   | A070 FT to FL430                                      | A066 FT |
+| Yankee        | Western Lane and Avon Lane                 | 35 NM to 50 NM   | A070 FT to FL430                                      | A071 FT |
+| Zulu          | Avon Lane and Northern Lane                | 35 NM to 50 NM   | A070 FT to FL430                                      | A071 FT |
+
+## Departure Procedures
+
+### IFR Climb
+
+On departure from ESL or WSL, pilots may request an IFR climb to penetrate weather en route to a training area. During the day, unless being vectored, pilots are responsible for terrain clearance below LSALT. Aircraft cleared for an "IFR CLIMB" must continue climbing, not above their cleared level, and report "CANCEL IFR" when able to resume VFR.
```

Comment:
I'm happy if we want to remove this section, and instead rely on pilots to file either IFR or VFR as appropriate, upgrading/downgrading the service required based on the weather conditions